### PR TITLE
add new parser - Detect-secrets

### PR DESCRIPTION
**Pull request by ansidorov**
detect-secrets is an aptly named module for (surprise, surprise) detecting secrets within a code base.

It could be found at: https://github.com/Yelp/detect-secrets

### CODE REVIEW

**Review by valentijnscholten — APPROVED** [review]
Can you check if we need to set dedupe algorithm to HASH_CODE in settings? And if we need a hash code field config or can use the default? (Might be good to check the same for the previous parsers by you which we already merged)

**Comment by ansidorov** [comment]
> Can you check if we need to set dedupe algorithm to HASH_CODE in settings? And if we need a hash code field config or can use the default? (Might be good to check the same for the previous parsers by you which we already merged)

If I understand correctly, we need to use the default for detect-secrets parser.

**Review by devGregA — APPROVED** [review]

File: docs/content/en/integrations/import.md [hunk 1]
```diff
@@ -668,6 +672,7 @@ The CSV output from the UI is now also accepted.
 
 ### Visual Code Grepper (VCG)
 
+
```
Comment (damiencarol) [inline docs/content/en/integrations/import.md:675]:
```suggestion

```

Comment (damiencarol) [reply docs/content/en/integrations/import.md:675]: Why this line.

File: dojo/unittests/scans/detect_secrets/no_findings.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    {
+      "path": "detect_secrets.filters.heuristic.is_sequential_string"
+    },
+    {
+      "path": "detect_secrets.filters.heuristic.is_templated_secret"
+    }
+  ],
+  "results": {},
+  "generated_at": "2021-05-19T10:45:24Z"
```
Comment (damiencarol) [inline dojo/unittests/scans/detect_secrets/no_findings.json:93]: Could you use this attribute to set the `date` attribute of findings?

Comment (ansidorov) [reply dojo/unittests/scans/detect_secrets/no_findings.json:93]: Done

**Review by damiencarol — APPROVED** [review]

**Pull request merged**